apps/wallet-mobile/src/features/Send/useCases/ConfirmTx/FailedTx/FailedTxScreen.tsx

```diff
@@ -24,14 +23,14 @@ export const FailedTxScreen = () => {
 
       <Spacer height={22} />
 
-      <Button onPress={resetAndGoToStart} title={strings.failedTxButton} style={styles.button} shelleyTheme />
+      <Button onPress={resetAndGoToStartTx} title={strings.failedTxButton} style={styles.button} shelleyTheme />
     </View>
   )
 }
 
 const useResetAndGoToStartTx = () => {
   const navigation = useNavigation()
-  const resetAndGoToStart = () =>
+  const resetAndGoToStartTx = () =>
```

Comment:
@jemubm there is a `navigation.ts` -> in the `common` maybe move it there? (I also believe you can import the reset from the main navigation, on top of my head)